packages/next/src/client/components/redirect.ts

```diff
@@ -88,3 +106,13 @@ export function getRedirectTypeFromError<U extends string>(
 
   return error.digest.split(';', 3)[1] as RedirectType
 }
+
+export function getRedirectStatusCodeFromError<U extends string>(
```

Comment:
Since this function is only used in app-render it's likely better for bundle size to move this into a separate file.